src/Components/Facility/ConsultationDetails.tsx

```diff
@@ -14,228 +14,348 @@ import { DailyRoundsModel } from "../Patient/models";
 
 const symptomChoices = [...SYMPTOM_CHOICES];
-const patientCategoryChoices = [...PATIENT_CATEGORY]
+const patientCategoryChoices = [...PATIENT_CATEGORY];
 
 export const ConsultationDetails = (props: any) => {
-    const { facilityId, patientId, consultationId } = props;
-    const dispatch: any = useDispatch();
-    const [isLoading, setIsLoading] = useState(false);
-    const [isDailyRoundLoading, setIsDailyRoundLoading] = useState(false);
-    const [consultationData, setConsultationData] = useState<ConsultationModel>({});
-    const [dailyRoundsListData, setDailyRoundsListData] = useState<Array<DailyRoundsModel>>([]);
-    const [totalCount, setTotalCount] = useState(0);
-    const [offset, setOffset] = useState(0);
-    const [currentPage, setCurrentPage] = useState(1);
-    const limit = 5;
+  const { facilityId, patientId, consultationId } = props;
+  const dispatch: any = useDispatch();
+  const [isLoading, setIsLoading] = useState(false);
+  const [isDailyRoundLoading, setIsDailyRoundLoading] = useState(false);
+  const [consultationData, setConsultationData] = useState<ConsultationModel>(
+    {}
+  );
+  const [dailyRoundsListData, setDailyRoundsListData] = useState<
+    Array<DailyRoundsModel>
+  >([]);
+  const [totalCount, setTotalCount] = useState(0);
+  const [offset, setOffset] = useState(0);
+  const [currentPage, setCurrentPage] = useState(1);
+  const limit = 5;
 
-    const fetchData = useCallback(
-        async (status: statusType) => {
-            setIsLoading(true);
-            const res = await dispatch(getConsultation(consultationId));
-            if (!status.aborted) {
-                if (res && res.data) {
-                    const data: ConsultationModel = {
-                        ...res.data,
-                        symptoms_text: '',
-                        category: patientCategoryChoices.find(i => i.id === res.data.category)?.text || res.data.category,
-                    }
-                    if (res.data.symptoms && res.data.symptoms.length) {
-                        const symptoms = res.data.symptoms.filter((symptom: number) => symptom !== 9).map((symptom: number) => {
-                            const option = symptomChoices.find(i => i.id === symptom);
-                            return option ? option.text.toLowerCase() : symptom;
-                        });
-                        data.symptoms_text = symptoms.join(', ');
-                    }
-                    setConsultationData(data);
-                }
-                setIsLoading(false);
-            }
-        },
-        [consultationId, dispatch]
-    );
+  const fetchData = useCallback(
+    async (status: statusType) => {
+      setIsLoading(true);
+      const res = await dispatch(getConsultation(consultationId));
+      if (!status.aborted) {
+        if (res && res.data) {
+          const data: ConsultationModel = {
+            ...res.data,
+            symptoms_text: "",
```

Comment:
Codacy found an issue: [Identifier 'symptoms_text' is not in camel case.](https://app.codacy.com/gh/coronasafe/care_fe/pullRequest?prid=5808160)